.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -14,20 +14,20 @@ Put an `x` into the box if that apply:
 <!--
 Briefly describe what this pull request does. Be sure to direct your reviewers'
 attention to anything that needs special consideration.
-We love pull requests that resolve an open renovate-config issue. If yours does, you
+We love Pull Requests that resolve an open renovate-config issue. If yours does, you
```

Comment:
pull requests is not capitalized https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests